spec/features/when_nomis_is_missing_information_spec.rb

```diff
@@ -83,6 +83,9 @@
           latestBookingId: booking_id
         }]
 
+        stub_request(:get, "#{stub_api_host}/staff/#{staff_id}").
+            to_return(status: 200, body: { staffId: staff_id, firstName: "TEST", lastName: "MOIC" }.to_json)
```

Comment:
nit: consistent indentation
```suggestion
          to_return(status: 200, body: { staffId: staff_id, firstName: "TEST", lastName: "MOIC" }.to_json)
```

Reply:
Indentation fixed